exe/server/main.go

```diff
@@ -64,15 +64,20 @@ func main() {
 		if err != nil {
 			log.Fatal(err)
 		}
-	}
 
-	repoPath := config.GetString("repopath")
-	if repoPath == "${HOME}/.texfc" {
-		home, err := os.UserHomeDir()
+		repoPath := config.GetString("repopath")
+		if repoPath == "${HOME}/.texfc" {
```

Comment:
Should `~` also be considered? https://github.com/mitchellh/go-homedir can be handy.

Reply:
Left this in my mind and forgot to comment.
Do you mean having the default be `~/.texfc` and using `go-homedir`?
That sounds pretty cool, will add that change to the next changes.